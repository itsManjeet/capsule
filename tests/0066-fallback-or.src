arr := [10, 203];

println(arr[0] or 55);
println(arr[1] or 55);
println(arr[2] or 55);
println(arr[1000] or 55);