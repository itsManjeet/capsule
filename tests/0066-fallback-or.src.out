10
203
55
55
